### PR TITLE
Skeleton for PDF Generation, About Page, other things

### PR DESCRIPTION
**Pull request by chaneberg**
Pull request includes:
- Skeleton for PDF Generation
- 'About' page
- Changed night 1 and night 2 to wednesday and thursday
- Made landing page get from new play lists: `wednesday-chronological` and `thursday-chronological`
- Question mark displayed for index of fuzzed rankings
- added a way to see all reviews in admin site
- added "Plays ranked by review positivity:" on play list page

PDF Generation really should be behind Django's admin authorization. For now I put this in the URL routing:

<pre>+    # urls that look like 'directors-report/' are handled here
+    # uncomment this line to test director's report page
+    # comment this out to prevent the public from viewing the director's report page
+    # TODO: put this behind admin authorization
+    #url(r'^directors-report/$', views.getDirectorsReport, name='directors-report')</pre>

The HTML that is served is unstyled and doesn't even have `<html>` and `<body>` tags.

### CODE REVIEW

**Comment by chaneberg** [comment]
There it is:

capture

**Comment by mnijmeh11** [comment]
why pdf

**Comment by chaneberg** [comment]
@mnijmeh11 It is a report they can easily print out or save on their computer.

**Comment by bencdelaney** [comment]
This is looking good, but I don't think it should live in the API. We can access the database identically from all apps. An app called `statler_util` could be good, and we could include the human review elements there as well. This may be a little more verbose than we need; the directors are probably more concerned with the review text than with metadata.

**Comment by chaneberg** [comment]
@B3nD3lan3y,

I will make the statler_util app and change url routing.

I handed this off to Moses, so could you comment on #146 to tell him which data fields should be removed.

**Comment by chaneberg** [comment]
I added an About page

capture

**Comment by chaneberg** [comment]
@B3nD3lan3y I made the statler_util app and changed the url routing.

**Comment by chaneberg** [comment]
I changed night 1 and night 2 to wednesday and thursday and made landing page get from new play lists: wednesday-chronological and thursday-chronological.

File: statler_site/templates/about.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{% block main-content %}
+    <div class="container">
+      <div class="about-page-text">
+        <h1>About <i>Ten! Ten! Ten!</i></h1>
+        <p>
+          <i>Ten! Ten! Ten!</i> is the final directing project for THEA 370: Directing of the Stage. <i>Ten! Ten! Ten!</i> has been a theatre tradition at George Fox University for the past 15 years. For more information about GFU Theatre go to <a href="http://theatre.georgefox.edu">theatre.georgefox.edu</a>.
+        </p>
+        <h1>About the CABAM Team</h1>
+        <p>
+          tententenreviews.com is the product of a semester-long project for GFU's Software Engineering class. Our task was to create a system that would rank products like movies or plays based on a sentiment analysis of user reviews, instead of a numbered ranking system like "stars." We thought it would be a great learning experience (and a fun challenge!) for us to make the system a real product used by real people, so we contacted the Theater Department to see if they would like to be our client. For some reason, they did, and we're thrilled to present to you a site specially designed for <i>Ten! Ten! Ten!</i> reviews.
```
Comment (bencdelaney) [inline statler_site/templates/about.html:12]: Breaking these lines (at 80? 120?) May make things more readable.

**Comment by bencdelaney** [comment]
Looks good.

**Comment by mnijmeh11** [comment]
I didn't know that you wanted me to review it immediately.

**Comment by mnijmeh11** [comment]
the stuff looks good and I can be there for the human review time....

**Pull request merged**